tinygrad/tensor.py

```diff
@@ -170,7 +170,7 @@ def ones(*shape, **kwargs): return Tensor.full(argfix(*shape), 1, **kwargs)
   @staticmethod
   def arange(start, stop=None, step=1, **kwargs):
     if stop is None: stop, start = start, 0
-    return Tensor.full((math.ceil((stop-start)/step),), step, **kwargs).cumsum() + (start - step)
+    return Tensor([i for i in range(start, stop, step)], **kwargs)
```

Comment:
Yea, I can't merge this, it's a CPU copy now. I'll add fixing the N^2 arange as a $500 bounty

Reply:
N is the number of unique tokens: 51864. I've seen arrays of size (N, 2N) getting allocated, which works out to around 20GB

Reply:
After reverting this change, test_whisper is broken on torch CI again: `RuntimeError: [enforce fail at alloc_cpu.cpp:83] err == 0. DefaultCPUAllocator: can't allocate memory: you tried to allocate 21519203420 bytes. Error code 12 (Cannot allocate memory)`. Merging something like #2207 may be helpful in the short term! (until someone figures out the O(N) impl)